src/ui/view/dataverse/component/itemOverview/itemOverview.tsx

```diff
@@ -0,0 +1,48 @@
+import type { FC } from 'react'
+import { useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+import classNames from 'classnames'
+import { Tags } from '@/ui/view/dataverse/component//tags/tags'
+import './itemOverview.scss'
+
+type ItemOverviewProps = {
+  type: DataverseItem
+  title: string
+  description: string
+  tags: string[]
+}
+type DataverseItem = 'dataspace' | 'dataset' | 'service'
+type ColorVariant = 'primary-color' | 'primary-color-variant-3' | 'primary-color-variant-4'
+
+const ItemOverview: FC<ItemOverviewProps> = ({ type, title, description, tags }): JSX.Element => {
+  const { t } = useTranslation('common')
+
+  const renderItemTypeColor = useCallback((type: DataverseItem): ColorVariant => {
```

Comment:
Here's a duplicate too

Reply:
I will do a custom hook for this. However it results to code refactoring, which is what we want to avoid when we create a new feature.

Reply:
yes but we don't know when we will have the time to make the refactor, so as it is linked to this feature we wand to avoid adding duplicated code.